dotnetv3/S3/README.md

```diff
@@ -0,0 +1,92 @@
+# Amazon Simple Storage Service (Amazon S3) code examples in C#
+
+## Purpose
+
+Customers of all sizes and industries can use Amazon S3 to store and protect any amount of data for a range of use cases, such as data lakes, websites, mobile applications, backup and restore, archive, enterprise applications, IoT devices, and big data analytics. Amazon S3 provides management features so that you can optimize, organize, and configure access to your data to meet your specific business, organizational, and compliance requirements.
```

Comment:
Amazon S3 -> Amazon Simple Storage Service (Amazon S3)

Reply:
changed but I think that the title should count as the first use.

Reply:
You're right, my mistake